src/jaxspec/model/additive.py

```diff
@@ -389,3 +383,75 @@ def integrand(kT, e, tin, p):
         integral = integrate_interval(integrand)
 
         return norm * 2.78e-3 * (0.75 / p) / tin * jax.vmap(lambda e: integral(tout, tin, e, tin, p))(energy)
+
+
+class Agauss(AdditiveComponent):
```

Comment:
About this class : I think the idea here would be to still get the energy as entry, `jaxspec` is standardized to work with spectra computed in keV. You should take the same template as in Gauss but simply add conversion factors to the parameters, by this I mean that the central wavelength should be converted to the central energy of the line and then passed to the gaussian pdf function

Reply:
Changes added: energy as entry + conversion factor hc